project-management-application-web/src/app/auth/auth-routing.module.ts

```diff
@@ -2,7 +2,7 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
 const routes: Routes = [
-  {path:'',redirectTo:'login', pathMatch: 'full'}
+  {path:'',redirectTo:'/login', pathMatch: 'full'}
```

Comment:
being a child route, this should render a component not redirect from child to a main root